ops/gulp/docs-js.js

```diff
@@ -0,0 +1,27 @@
+const gulp = require('gulp');
+const gp_concat = require('gulp-concat');
+const gp_rename = require('gulp-rename');
+const gp_uglify = require('gulp-uglify');
+const gulpif = require('gulp-if');
+const gulp_jslint = require('gulp-jslint');
+
+let environment = require('../lib/environment');
+
+const paths = {
+    src: './src/scripts',
```

Comment:
Since this JS is for the site only and not part of the library, it should be in `docs` and not `src`.